VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/venue-list/venue-list.component.html

```diff
@@ -4,34 +4,19 @@ <h1 class="govuk-heading-m">{{'venue-list.title' | translate}}</h1>
   </div>
 
   <div class="govuk-grid-column-three-quarters">
-    <ng-select
-      id="venue-allocation-list"
-      class="custom"
-      [attr.aria-label]="'venue-list.allocation-list-label' | translate"
-      [items]="judges"
-      [multiple]="true"
-      bindLabel="name"
-      placeholder="Choose lists"
-      [(ngModel)]="selectedJudges"
-      [closeOnSelect]="false"
-      (change)="updateSelection()"
-      [clearSearchOnAdd]="true"
-      [selectOnTab]="true"
-    >
+    <ng-select id="venue-allocation-list" class="custom"
+      [attr.aria-label]="'venue-list.allocation-list-label' | translate" [items]="venues" [multiple]="true"
+      bindLabel="name" bindValue="name" placeholder="Choose lists" [(ngModel)]="selectedJudges" [closeOnSelect]="false"
```

Comment:
Can you rename the variable `selectedJudges` in `[(ngModel)]="selectedJudges"` to something more fitting

Reply:
I think this should be something like selectedCourtRooms?